### DISCUSSION TITLE
Bun runtime

### DISCUSSION THREAD

**Discussion by publicvirtualvoid** [moleculerjs/moleculer#1294] [General]
Is this supported? I ran some of the examples in Bun and it seemed to work. Is there any reason parts of Moleculer wouldn't work in BunJS?

**Comment by ImTheDeveloper** [comment]
I think some third party stuff didn't work for me I believe bull queue isn't working with bun as of yet so moleculer-bull isn't either. I had to use bee

**Comment by srcmayte** [comment]
What benefits have you seen by moving to bun?
